feature/aks-cluster-creation/terraform.tfvars

```diff
@@ -0,0 +1,34 @@
+existing_vnet_subnet_id = ""
+existing_resource_group_name = ""
+prefix = "ropu"
+location = "North Central US"
+kubernetes_version = "1.17.7"
+vnet_address_space = ["10.1.1.0/24"]
+aks_address_prefix = ["10.1.1.0/24"]
+public_ssh_key_path = "~/.ssh/id_rsa.pub"
+nodes_public_ip = false
+network_plugin = "azure"
+network_policy = "azure"
+linux_admin_username = "operation"
+default_node_pool = {
+    name                           = "linux"
+    node_count                     = 1
+    vm_size                        = "Standard_DS2_v2"
+    os_disk_size_gb                = "50"
+    type                           = "VirtualMachineScaleSets"
+    enable_auto_scaling            = false
+    min_count                      = null
+    max_count                      = null
+    }
+additional_cluster_node_pools = {
+    name                           = "windows"
+    node_count                     = 0
+    vm_size                        = "Standard_DS2_v2"
+    os_type                        = "Windows"

```

Comment:
why windows in defaults?

Reply:
This is additional node pool with 0 node count (because we would like to have different options). The default node pool has by default Linux since AKS need linux os :)
 Summing up: 
- Default node pool - Linux 
- Additional node pool - Windows/Linux

Reply:
Yes I got it but still think it should be linux as default. But approved.